pkg/trace/agent/agent.go

```diff
@@ -77,6 +79,9 @@ type Agent struct {
 
 	// Used to synchronize on a clean exit
 	ctx context.Context
+
+	// transformers is the list of transformers used by agent.Process()
```

Comment:
"...used by agent.Process()"

Why do we assume that the reader knows what agent.Process is? I'd rather describe that the transformers are meant to do, e.g. "...used to process incoming traces".